src/_locales/zh/messages.json

```diff
@@ -261,8 +261,10 @@
     "submit": "提交",
     "unknown_time": "未知时间",
     "username": "用戶名",
-    "version": "版本",
-    "version_link": "https://github.com/DimensionDev/Maskbook/releases#{{tag}}",
+    "version_of_stable": "版本 {{version}}",
+    "version_of_unstable": "版本 {{version}}-{{build}}-{{hash}}",
+    "version_of_release": "https://github.com/DimensionDev/Maskbook/releases/tag/{{tag}}",
```

Comment:
Why links need to be put in i18n file?

Reply:
#816

Reply:
But why? No design notes in that issue to explain